baus/datasources.py

```diff
@@ -46,13 +46,39 @@ def outputs_dir(run_setup):
 def viz_dir(run_setup):
     return os.path.join(run_setup['viz_dir'])
 
+@orca.injectable('sqft_per_job_adj_file', cache=True)
+def sqft_per_job_adj_file(run_setup):
+    # if no sqft_per_job_adj_file defined in yaml, return the default adjuster file
+    
+    adj_file = run_setup.get('sqft_per_job_adj_file', 'sqft_per_job_adjusters.csv')
+    return adj_file
+
+@orca.injectable('emp_reloc_rates_adj_file', cache=True)
+def emp_reloc_rates_adj_file(run_setup):
+    # if no emp_reloc_rates_adj_file defined in yaml, return the default adjuster file
+    
+    adj_file = run_setup.get('emp_reloc_rates_adj_file', 'employment_relocation_rates_overwrites.csv')
+    return adj_file
+
+
+@orca.injectable('elcm_spec_file', cache=True)
+def elcm_spec_file(run_setup):
+    # if no elcm_spec_file defined in yaml, return the default file
+    
+    elcm_file = run_setup.get('elcm_spec_file', 'elcm.yaml')
```

Comment:
I think this is a case where defaults are useful - if nothing is provided in the run_setup, just gets the plain vanilla specification. In a way just like the old hard-coding days of yore, which we still do for some data sources.

Reply:
@theocharides There are actually two questions folded into this.

1. Whether to have a default option for specifications
1. Whether to expose the specification in the yaml file at all.

The current approach takes the [hard coded](https://github.com/BayAreaMetro/bayarea_urbansim/blob/081b700190eb51a85276e73735c94d16a4223f56/baus/models.py#L29) approach.
This PR then exposes that file in the yaml, which in a way adds to transparency, but then partially walks it back by making the yaml keyword optional, with a default setting which is just the one in the `models.py` right now.
I am fine with shelving to a separate branch where we can address the general pattern of specification selection.